### PR TITLE
DOC: Typos found by codespell

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
Running [codespell](https://github.com/codespell-project/codespell) manually for now, but could be automated as suggested in https://github.com/systemd/systemd/issues/15436 if there is interest.

### CODE REVIEW

File: doc/source/gitwash/development_workflow.rst [hunk 1]
```diff
@@ -334,7 +334,7 @@ Rewriting commit history
 
    Do this only for your own feature branches.
 
-There's an embarassing typo in a commit you made? Or perhaps the you
+There's an embarrassing typo in a commit you made? Or perhaps the you
```
Comment (matthew-brett) [inline doc/source/gitwash/development_workflow.rst:337]: I just can't remember whether this was a joke or not.  But - oh well.

**Review by matthew-brett — APPROVED** [review]
Thanks - and yes - an automatic check would be good.

File: doc/source/gitwash/development_workflow.rst (hunk 1 above)
Comment (DimitriPapadopoulos) [reply doc/source/gitwash/development_workflow.rst:337]: Ah, didn't notice the context! Perhaps I can leave it then.

Comment (matthew-brett) [reply doc/source/gitwash/development_workflow.rst:337]: If you can be bothered, I enjoy the irony...

Comment (DimitriPapadopoulos) [reply doc/source/gitwash/development_workflow.rst]: Done

**Pull request merged**